tests/unit_tests/test_wave_01.py

```diff
@@ -49,7 +49,7 @@ def test_removing_not_found_is_false():
 
     result = vendor.remove(item)
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
+    assert result == False
```

Comment:
It would also be good to add an assert to make sure the length of the inventory didn't get changed.